examples/js/controls/TrackballControls.js

```diff
@@ -561,22 +561,14 @@ THREE.TrackballControls = function ( object, domElement ) {
 
 			case 0:
 				_state = STATE.NONE;
+				_touchZoomDistanceStart = _touchZoomDistanceEnd = 0;
 				break;
 
 			case 1:
 				_state = STATE.TOUCH_ROTATE;
```

Comment:
Ah, you are correct. Leaving it out works fine. Did some further testing and it looks like neither of the `_touchZoomDistanceStart = _touchZoomDistanceEnd = 0;` lines are needed (564 and 570) since the values will be set in the `touchstart` function when two or more touches are present.